tests/test_dynamics.py

```diff
@@ -867,7 +867,8 @@ def custom_dynamic(states, controls, parameters, nlp, with_contact=False) -> tup
         dq = DynamicsFunctions.compute_qdot(nlp, q, qdot)
         ddq = DynamicsFunctions.forward_dynamics(nlp, q, qdot, tau, with_contact)
 
-        return dq, ddq
+        defects = MX.zeros(1)
+        return (dq, ddq), defects
```

Comment:
Again, we have to have a better way for this

Reply:
yes